src/import-utils.ts

```diff
@@ -93,8 +105,13 @@ export async function inlineHtmlImport(
     return;
   }
 
-  // If the analyzer could not load the import document, we can't inline it, so
-  // lets skip it.
+  // We will stop short of actually inlining a `lazy-import` or its children.
```

Comment:
Updated comment:

```typescript
  // We don't actually inline a `lazy-import` because its loading is intended
  // to be deferred until the client requests it.
```